site/build/reference/render.js

```diff
@@ -148,18 +151,27 @@ function renderProperties(language, docs, templates) {
     var body = renderText(docs.properties[key], templates).content;
     var source = docs.source.properties[key];
     var id = escapeKey(property.key);
+    var template = templates.entry;
 
     if (uniqueIDs[id]) {
-      throw new Error('Duplicate html id for key: ' + property.key + ' and ' +
```

Comment:
I think this should be `' and '` to have the error look like `foo and bar` instead of `fooandbar`.